frontend/src/components/Sidebar/index.tsx

```
@@ -24,13 +36,13 @@ const DrawerList = (
       </List>
       <Divider />
       <List>
-        {['3DHome', 'Stock', 'Calculator'].map((text, index) => (
-          <span onClick={() => {navigate(text)}}><ListItem key={text} disablePadding>
+        {sidebarList.map( sidebarItem => (
+          <span onClick={() => {navigate(sidebarItem.path)}}><ListItem key={sidebarItem.label} disablePadding>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)